app/scripts/modules/kubernetes/serverGroup/configure/configuration.service.js

```diff
@@ -42,6 +56,10 @@ module.exports = angular.module('spinnaker.serverGroup.configure.kubernetes.conf
       return `*${query}*`;
     }
 
+    function grabImageAndTag(imageId) {
+      return imageId.split('/')[2];
```

Comment:
Valid image ID's include
`gcr.io/image:tag` <- that will fail here.
`gcr.io/org/image:tag`

Instead I recommend splitting by `'/'`, and taking the last item from the list.